modules/ROOT/pages/release-notes/release-notes52.adoc

```diff
@@ -115,31 +118,32 @@ The {productname} 5.2 release includes an accompanying release of the *Accessibi
 * To improve decorative image checks.
 * To accommodate the `a11y_advanced_options` setting added in {productname} 5.2.
 
-For information on the `a11y_advanced_options` setting, see: link:{baseurl}/plugins/premium/a11ychecker/#a11y_advanced_options[Accessibility Checker plugin].
+For information on the `a11y_advanced_options` setting, see: xref:plugins/premium/a11ychecker.adoc#a11y_advanced_options[Accessibility Checker plugin].
 
+[[checklist110]]
 === Checklist 1.1.0
 
 The {productname} 5.2 release includes an accompanying release of the *Checklist* premium plugin.
 
 *Checklist* 1.1.0 adds a new optional checklist menu item.
 
-For information on the checklist plugin, see: link:{baseurl}/plugins/premium/checklist/[Checklist plugin].
+For information on the checklist plugin, see: xref:plugins/premium/checklist.adoc[Checklist plugin].
 
 === Link Checker 2.1.0
 
 The {productname} 5.2 release includes an accompanying release of the *Link Checker* premium plugin.
 
 *Link Checker* 2.1.0 adds a new `linkchecker_preprocess` function for adjusting links before link checking.
 
-For information on the `linkchecker_preprocess` option, see: link:{baseurl}/plugins/premium/linkchecker/#linkchecker_preprocess[Link Checker plugin - `linkchecker_preprocess`].
+For information on the `linkchecker_preprocess` option, see: xref:plugins/premium/linkchecker.adoc#linkchecker_preprocess[Link Checker plugin - `linkchecker_preprocess`].
 
 === Page Embed 1.1.0
 
 The {productname} 5.2 release includes an accompanying release of the *Page Embed* premium plugin.
 
 *Page Embed* 1.1.0 adds a new `tiny_pageembed_inline_styles` setting which inlines all styles, while also keeping CSS classes, when rendering the embed content. This is useful when the additional CSS classes can't be added to your site.
 
-For information on inlining page embed styles, see: link:{baseurl}/plugins/premium/pageembed/#inlinestylesforpageembeds[Page Embed plugin - Inline styles for page embeds].
+For information on inlining page embed styles, see: xref:plugins/premium/pageembed.adoc#inlinestylesforpageembeds[Page Embed plugin - Inline styles for page embeds].
```

Comment:
This xref doesn't resolve as the `inlinestylesforpageembeds` id doesn't exist in `pageembed.adoc`. Now this appears to be broken on production as well, so based on the changes done to that file I believe it should be this instead.
```suggestion
For information on inlining page embed styles, see: xref:plugins/premium/pageembed.adoc#tiny_pageembed_inline_styles[Page Embed plugin - Inline styles for page embeds].
```
However, that will also need the id to be added to `tiny_pageembed_inline_styles.adoc`. e.g:
```
[[tiny_pageembed_inline_styles]]
=== `tiny_pageembed_inline_styles`
```

Reply:
done